### ISSUE TITLE
Does not reexport compiled proptypes

### ISSUE THREAD

**Issue by echenley** [brigand/babel-plugin-flow-react-proptypes#190]
## Example

```js
// path/form/index.js
// @flow

import ResponseForm from './response-form-container'
import type { ResponseModel, Props } from './response-form'

export type { ResponseModel, Props }
export default ResponseForm

```

```js
// path/form/response-form.js
// @flow
// ...

export type ResponseModel = {
  category_id: ?string,
  id?: string,
  name: string,
  content: string,
}

// ...
```

```js
// path/index.js
import type { ResponseModel } from './form'
```

This ends up throwing an error:

```
'./form' does not contain an export named 'bpfrpt_proptype_ResponseModel'
```

The problem is that `path/form/index.js` compiles to:

```js
import ResponseForm from './response-form-container';
var _default = ResponseForm;

export default _default;
import { bpfrpt_proptype_ResponseModel } from './response-form';
import { bpfrpt_proptype_Props } from './response-form';
```

You can see that the proptypes are not being exported. Not sure if this is an easy fix. I am running into this error after upgrading from 9.0.0 to 19.1.0, so quite the jump.

**Comment by echenley** [comment]
I was able to avoid the issue by using `export type`:

```js
// path/form/index.js
// @flow

import ResponseForm from './response-form-container'

export type { ResponseModel, Props } from './response-form'
export default ResponseForm
```

This results in:

```js
import ResponseForm from './response-form-container';

var _default = ResponseForm;

export default _default;
import { bpfrpt_proptype_ResponseModel } from './response-form';
export { bpfrpt_proptype_ResponseModel };
import { bpfrpt_proptype_Props } from './response-form';
export { bpfrpt_proptype_Props };
```

**Comment by brigand** [comment]
Thanks for the report, and definitely something that should be supported. I'll try to fix this in the next day or two.

**Comment by brigand** [comment]
Published a partial fix in 20.0.0. You can't rename the types, but otherwise it should work.
